pkg/sdk/config_test.go

```diff
@@ -25,116 +35,302 @@ func TestLoadConfigFile(t *testing.T) {
 	password='abcd1234'
 	role='SECURITYADMIN'
 	`
-	configPath := testFile(t, "config", []byte(c))
-	t.Setenv(snowflakeenvs.ConfigPath, configPath)
+	configPath := testhelpers.TestFile(t, "config", []byte(c))
 
-	m, err := loadConfigFile()
+	m, err := loadConfigFile(configPath)
 	require.NoError(t, err)
-	assert.Equal(t, "TEST_ACCOUNT", m["default"].Account)
-	assert.Equal(t, "TEST_USER", m["default"].User)
-	assert.Equal(t, "abcd1234", m["default"].Password)
-	assert.Equal(t, "ACCOUNTADMIN", m["default"].Role)
-	assert.Equal(t, "TEST_ACCOUNT", m["securityadmin"].Account)
-	assert.Equal(t, "TEST_USER", m["securityadmin"].User)
-	assert.Equal(t, "abcd1234", m["securityadmin"].Password)
-	assert.Equal(t, "SECURITYADMIN", m["securityadmin"].Role)
+	assert.Equal(t, "TEST_ACCOUNT", *m["default"].Account)
+	assert.Equal(t, "TEST_USER", *m["default"].User)
+	assert.Equal(t, "abcd1234", *m["default"].Password)
+	assert.Equal(t, "ACCOUNTADMIN", *m["default"].Role)
+	assert.Equal(t, "TEST_ACCOUNT", *m["securityadmin"].Account)
+	assert.Equal(t, "TEST_USER", *m["securityadmin"].User)
+	assert.Equal(t, "abcd1234", *m["securityadmin"].Password)
+	assert.Equal(t, "SECURITYADMIN", *m["securityadmin"].Role)
 }
 
 func TestProfileConfig(t *testing.T) {
-	c := `
+	rsaPrivateKey, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
nit: this setup (private key creation etc) could be extracted for readability

Reply:
Done.